source/ouibounce.js

```diff
@@ -82,7 +82,7 @@ function ouibounce(el, custom_config) {
   // You can use ouibounce without passing an element
   // https://github.com/carlsednaoui/ouibounce/issues/30
   function fire() {
-    if (isDisabled()) { return; }
+    if (isDisabled() || supressed) { return; }
```

Comment:
IMO, supressed should be a part of the isDisabled function. It's easier to read that way.

```
function isDisabled() {
    return supressed || (checkCookieValue(cookieName, 'true') && !aggressive);
  }
```